conf/vertebrates/production_reg_conf.pl

```diff
@@ -84,61 +84,61 @@
     # 'compara_curr_3'   => [ 'mysql-ens-compara-prod-3', "ensembl_compara_$curr_release" ],
 
     # homology dbs
-    'compara_members'         => [ 'mysql-ens-compara-prod-1',  'carlac_vertebrates_load_members_104' ],
-    'compara_ptrees'          => [ 'mysql-ens-compara-prod-1',  'carlac_default_vertebrates_protein_trees_104' ],
-    'compara_nctrees'         => [ 'mysql-ens-compara-prod-5',  'carlac_default_vertebrates_ncrna_trees_104' ],
-    'murinae_ptrees'          => [ 'mysql-ens-compara-prod-2',  'carlac_vertebrates_murinae_protein_reindexed_trees_104' ],
-    'murinae_nctrees'         => [ 'mysql-ens-compara-prod-2',  'carlac_vertebrates_murinae_ncrna_reindexed_trees_104' ],
-    'murinae_ptrees_prev'     => [ 'mysql-ens-compara-prod-8',  'jalvarez_murinae_vertebrates_protein_trees_103' ],
-    'murinae_nctrees_prev'    => [ 'mysql-ens-compara-prod-7',  'jalvarez_murinae_vertebrates_ncrna_trees_103' ],
-    'pig_breeds_ptrees'       => [ 'mysql-ens-compara-prod-3',  'carlac_vertebrates_pig_breeds_protein_reindexed_trees_104' ],
-    'pig_breeds_nctrees'      => [ 'mysql-ens-compara-prod-3',  'carlac_vertebrates_pig_breeds_ncrna_reindexed_trees_104' ],
-    'pig_breeds_ptrees_prev'  => [ 'mysql-ens-compara-prod-4',  'jalvarez_pig_breeds_vertebrates_protein_trees_103' ],
-    'pig_breeds_nctrees_prev' => [ 'mysql-ens-compara-prod-10', 'jalvarez_pig_breeds_vertebrates_ncrna_trees_103' ],
+    #'compara_members'         => [ 'mysql-ens-compara-prod-1',  'carlac_vertebrates_load_members_104' ],
+    #'compara_ptrees'          => [ 'mysql-ens-compara-prod-1',  'carlac_default_vertebrates_protein_trees_104' ],
+    #'compara_nctrees'         => [ 'mysql-ens-compara-prod-5',  'carlac_default_vertebrates_ncrna_trees_104' ],
+    #'murinae_ptrees'          => [ 'mysql-ens-compara-prod-2',  'carlac_vertebrates_murinae_protein_reindexed_trees_104' ],
+    #'murinae_nctrees'         => [ 'mysql-ens-compara-prod-2',  'carlac_vertebrates_murinae_ncrna_reindexed_trees_104' ],
+    'murinae_ptrees_prev'     => [ 'mysql-ens-compara-prod-2',  'carlac_vertebrates_murinae_protein_reindexed_trees_104' ],
+    'murinae_nctrees_prev'    => [ 'mysql-ens-compara-prod-2',  'carlac_vertebrates_murinae_ncrna_reindexed_trees_104' ],
+    #'pig_breeds_ptrees'       => [ 'mysql-ens-compara-prod-3',  'carlac_vertebrates_pig_breeds_protein_reindexed_trees_104' ],
+    #'pig_breeds_nctrees'      => [ 'mysql-ens-compara-prod-3',  'carlac_vertebrates_pig_breeds_ncrna_reindexed_trees_104' ],
+    'pig_breeds_ptrees_prev'  => [ 'mysql-ens-compara-prod-3',  'carlac_vertebrates_pig_breeds_protein_reindexed_trees_104' ],
+    'pig_breeds_nctrees_prev' => [ 'mysql-ens-compara-prod-3',  'carlac_vertebrates_pig_breeds_ncrna_reindexed_trees_104' ],
```

Comment:
You are likely going to initialise all of these, so on top of commenting them out, you could empty them to leave as templates to fill - this makes it easier to remember to update it too:
```suggestion
    #'compara_members'         => [ 'mysql-ens-compara-prod-1',  '' ],
    #'compara_ptrees'          => [ 'mysql-ens-compara-prod-X',  '' ],
    #'compara_nctrees'         => [ 'mysql-ens-compara-prod-X',  '' ],
    #'murinae_ptrees'          => [ 'mysql-ens-compara-prod-X',  '' ],
    #'murinae_nctrees'         => [ 'mysql-ens-compara-prod-X',  '' ],
    'murinae_ptrees_prev'     => [ 'mysql-ens-compara-prod-2',  'carlac_vertebrates_murinae_protein_reindexed_trees_104' ],
    'murinae_nctrees_prev'    => [ 'mysql-ens-compara-prod-2',  'carlac_vertebrates_murinae_ncrna_reindexed_trees_104' ],
    #'pig_breeds_ptrees'       => [ 'mysql-ens-compara-prod-X',  '' ],
    #'pig_breeds_nctrees'      => [ 'mysql-ens-compara-prod-X',  '' ],
    'pig_breeds_ptrees_prev'  => [ 'mysql-ens-compara-prod-3',  'carlac_vertebrates_pig_breeds_protein_reindexed_trees_104' ],
    'pig_breeds_nctrees_prev' => [ 'mysql-ens-compara-prod-3',  'carlac_vertebrates_pig_breeds_ncrna_reindexed_trees_104' ],
```